### PR TITLE
Statistics: Send TEvAnalyzeTable to shards

### PR DESCRIPTION
**Pull request by azevaykin**
### Changelog entry 

...

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/core/statistics/aggregator/aggregator_impl.cpp [hunk 1]
```diff
@@ -856,7 +920,8 @@ bool TStatisticsAggregator::OnRenderAppHtmlPage(NMon::TEvRemoteHttpInfo::TPtr ev
             str << "PendingRequests: " << PendingRequests.size() << Endl;
             str << "ProcessUrgentInFlight: " << ProcessUrgentInFlight << Endl << Endl;
 
-            str << "TraversalTableId: " << TraversalTableId << Endl;
+            str << "TraversalPathId: " << TraversalPathId << Endl;
+            str << "TraversalPathId: " << TraversalPathId << Endl;
```
Comment (alexd65536) [inline ydb/core/statistics/aggregator/aggregator_impl.cpp:924]: повтор строчки

Comment (azevaykin) [reply ydb/core/statistics/aggregator/aggregator_impl.cpp]: Поправил

File: ydb/core/statistics/aggregator/tx_response_tablet_distribution.cpp [hunk 2]
```diff
@@ -67,15 +77,13 @@ struct TStatisticsAggregator::TTxResponseTabletDistribution : public TTxBase {
...
         }
 
-        ++Self->TraversalRound;
-        ++Self->GlobalTraversalRound;
-        Self->PersistGlobalTraversalRound(db);
-        outRecord.SetRound(Self->GlobalTraversalRound);
         Action = EAction::SendAggregate;
+        return ExecuteStartForceTraversal(txc);
 
         return true;
```
Comment (alexd65536) [inline ydb/core/statistics/aggregator/tx_response_tablet_distribution.cpp:88]: лишний return true;

Comment (azevaykin) [reply ydb/core/statistics/aggregator/tx_response_tablet_distribution.cpp]: Поправил

**Review by alexd65536 — APPROVED** [review]

**Pull request merged**